docs/guides/noise-learning.ipynb

```diff
@@ -0,0 +1,194 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1b2370f0-a4ee-49b1-9e53-e4527d37bc6f",
+   "metadata": {},
+   "source": [
+    "# Noise learning helper\n",
+    "\n",
+    "The error mitigation techniques [TREX](./error-mitigation-and-suppression-techniques#twirled-readout-error-extinction-trex) and [PEC](./error-mitigation-and-suppression-techniques#probabilistic-error-cancellation-pec) both utilize a noise learning component based on a [Pauli-Lindblad noise model](https://www.nature.com/articles/s41567-023-02042-2) which is typically managed during execution after submitting one or more jobs through `qiskit-ibm-runtime`. However, as of `qiskit-ibm-runtime` 0.27.1 and later, a [`NoiseLearner`](../api/qiskit-ibm-runtime/noise_learner) and a `NoiseLearnerOptions` class for executing and storing the results of these noise learning experiments now exists. This page provides an overview of its usage and the associated options available.\n",
+    "\n",
+    "\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "The `NoiseLearner` class performs experiments which characterizes noise processes based on a Pauli-Lindblad noise model for one (or more) circuits. It possesses a `run()` method which will execute the learning experiments and takes as input either a list of circuits or a [PUB](./primitive-input-output#overview-of-pubs) and returns a `NoiseLearnerResult` containing the learned noise channels and metadata about the job(s) submitted. Below is a code snippet demonstrating the usage of the helper program."
```

Comment:
Add reference for Pauli-Lindblad noise mdoel: https://arxiv.org/abs/2201.09866

Reply:
Done o7